packages/node/src/clients/external-match/index.ts

```diff
@@ -43,8 +43,12 @@ export class ExternalMatchClient {
         chainId: number;
         apiKey: string;
         apiSecret: string;
+        overrides?: Partial<SDKConfig>;
     }) {
-        const configv2 = getSDKConfig(params.chainId);
+        const defaultConfig = getSDKConfig(params.chainId);
+        const configv2 = params.overrides
+            ? { ...defaultConfig, ...params.overrides }
```

Comment:
do we need to match on `params.overrides ? ...` here? if `params.override === undefined`, we can still safely do `{ ...defaultConfig, ...params.overrides }`

Reply:
while this compiles, technically the spread operator is only allowed to be used on object types, which undefined is not a subtype of.